### DISCUSSION TITLE
I want to click at the end of the video, how do I do that?

### DISCUSSION THREAD

**Discussion by TanMink** [AutomaApp/automa#162] [Q&A] [answered]
I want to click at the end of the video, how do I do that?
![Capture](https://user-images.githubusercontent.com/95482331/144709579-a480acec-3d16-46f1-9c85-d028c44bd46c.PNG)

**Comment by Kholid060** [comment, marked as answer]
It's not possible to do this task using Automa because Automa can only click an element.
